crates/lib/src/bootc_composefs/boot.rs

```diff
@@ -132,6 +132,8 @@ const SYSTEMD_LOADER_CONF_PATH: &str = "loader/loader.conf";
 const INITRD: &str = "initrd";
 const VMLINUZ: &str = "vmlinuz";
 
+const BOOTC_AUTOENROLL_PATH: &str = "usr/lib/bootc/keys";
```

Comment:
If these are only copied from the container to the bootloader state at install time, then I think this should be `/usr/lib/bootc/install/secureboot-keys` or so.

(Note `install/` subdir)

Also can you add a section to the install manpage about this?